promscale/index.md

```diff
@@ -5,14 +5,30 @@ product: promscale
 ---
 
 # Promscale
-Promscale is the open source observability backend for metrics and traces
-powered by SQL.
+Promscale is a unified metric and trace observability backend for Prometheus,
+Jaeger and OpenTelemetry built on PostgreSQL and TimescaleDB.
 
-It is built on top of PostgreSQL and TimescaleDB. Promscale supports Prometheus metrics and is 100% compliant with PromQL and OpenTelemetry traces.
-Its full SQL capabilities enable developers to correlate metrics, traces, and business data to get valuable insights.
-This is not possible when data is siloed in different systems.
+Promscale serves as a robust and 100% PromQL-compliant Prometheus remote
+storage and as a durable and scalable Jaeger storage backend. Promscale is a
+certified Jaeger storage backend.
 
-*   [Quick start Promscale][quick-start] get started with Promscale in no time.
+Unlike other observability backends, it has a simple and easy-to-manage
+architecture with just two components: the Promscale Connector and the
+Promscale Database (PostgreSQL with the TimescaleDB and Promscale extensions).
+
+<highlight type="cloud" header="Promscale with Timescale Cloud" button="Get started for free"
+to="https://console.cloud.timescale.com/signup?campaign=promscale&source=ps-docs-home">
+The best way to use Promscale is with Timescale Cloud, our fully hosted and managed
+database platform. Enjoy all the best features of Promscale without the
+hassle of managing your database—with automatic backups and failover, high
+availability, flexible scaling, security and data compliance, VPC peering, and
+much more. Get cost savings up to 94% compared to managed Prometheus offerings from
+AWS, GCP and Grafana.
```

Comment:
```suggestion
AWS, Google Cloud Platform, and Grafana.
```

Reply:
I don't think we need to expand GCP in the same way we don't expand AWS.

Reply:
In that case, we'll need to update our linter.